db.json

```diff
@@ -3826,7 +3913,8 @@
     "extensions": ["igx"]
   },
   "application/vnd.microsoft.portable-executable": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["exe","dll","cpl","drv","scr","efi","ocx","sys","lib"]
```

Comment:
https://www.iana.org/assignments/media-types/application/vnd.microsoft.portable-executable only lists .exe and .dll 

Long commit message at https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/d4c2ed1b62b51fed1f9fa003de6cfdc6a82df729

I think .lib may be misguided - they are COFF files on Windows.

It looks like they are getting that long list of file extensions from the infobox on http://fileformats.archiveteam.org/wiki/Portable_Executable